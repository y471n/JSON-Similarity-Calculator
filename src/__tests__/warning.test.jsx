import React from "react";
import { shallow } from "enzyme";

import Warning from "../components/warning";

it("Should render a span with given title", () => {
  const wrapper = shallow(<Warning title="hello" />);
  expect(wrapper).toMatchSnapshot();
  const span = wrapper.find("span");
  expect(span).toBeDefined();
  const result = span.text();
  expect(result).toBe("hello");
});
