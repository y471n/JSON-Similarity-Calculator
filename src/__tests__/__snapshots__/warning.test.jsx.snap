// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render a span with given title 1`] = `ShallowWrapper {}`;
